content/en/docs/reference/command-line-tools-reference/feature-gates/contextual-logging.md

```diff
@@ -9,6 +9,10 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.24"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.30"
 ---
-When you enable this feature gate, Kubernetes components that support
- contextual logging add extra detail to log output.
+When you disable this feature gate, Kubernetes components that support
+ contextual logging might produce less detailed log output because
+ some extra information might be provided only as part of the context.
```

Comment:
Note that I switched from "if you enable" to "if you disable" here because that is what users would do now.

Also note that when converting existing log calls we try to avoid information loss should the feature get disabled. But in new code that I recently wrote for DRA, I am assuming that the feature remains enabled and avoid logging information both via context and as parameters.

I expect this to happen more often going forward because the duplication is annoying.

Reply:
We always write descriptions as what happens when the feature is _enabled_.
Try
```markdown
Enables additional detail in log output from Kubernetes components that
support writing structure logs.
```

Reply:
Think about the graduation to stable; Kubernetes still has the feature, but now you cannot disable it. We want a feature description that makes sense for beta and for stable.

Reply:
Makes sense. I used the text that I had in the main file because this works regardless of the log output format.

> Enables extra details in log output of Kubernetes components that support
> contextual logging.